README.md

```diff
@@ -1,158 +1,111 @@
 
-1. The Data Importer Pod streams the file from the remote data store to the mounted backing storage volume. When the copy completes the importer pod terminates. The destination file name is always _disk.img_ but, since there is one volume per image file, the parent directory will (and must) differ.
+Expected file formats are:
 
-### Components
+- .tar
+- .gz
+- .img
+- .iso
+- .qcow2
 
-**Data Import Controller:** Long-lived Controller pod in "golden" namespace.
-The controller scans for "golden" PVCs in the same namespace looking for specific
-annotations:
-- kubevirt.io/storage.import.endpoint:  Defined by the admin: the full endpoint URI for the source file/image
-- kubevirt.io/storage.import.secretName: Defined by the admin: the name of the existing Secret containing the credential to access the endpoint.
-- kubevirt.io/storage.import.status: Added by the controller: the current status of the PVC with respect to the import/copy process. Values include:  ”In process”, “Success”, “ Failed”
+## Deploying CDI
 
-On detecting a new PVC with the endpoint annotation (and lacking the status annotation), the controller creates the Data Importer pod "golden" namespace. The controller performs clean up operations after the data import process ends.
+### Assumptions
+- A running Kubernetes cluster
+- A storage class and provisioner.
+- An HTTP or S3 file server hosting VM images
+- A namespace acting as the image registry (`default` is fine for tire kicking)
 
-**Data Import Pod:** Short-lived pod in "golden" namespace. The pod is created by the controller and consumes the secret (if any) and the endpoint annotated in the PVC. It copies the object referenced by the PVC's endpoint annotation to the destination directory used by the storage class/provider. In all cases the target file **name** is _disk.img_.
+### Either clone this repo or download the necessary manifests directly:
 
-**Dynamic Provisioner:** Existing storage provisoner(s) which create the Golden Persistent Volume that reference an empty cluster storage volume. Creation begins automatically when the Golden PVC is created by an admin.
+`$ git clone https://github.com/kubevirt/containerized-data-importer.git`
 
-**Endpoint Secret:** Long-lived secret in "golden" namespace that is defined and created by the admin. The Secret must contain the access key id and secret key required to make requests from the object store. The Secret is mounted by the Data Import pod.
+*Or*
 
-**"Golden" Namespace:** Restricted/private Namespace for Golden PVCs and endpoint Secrets. Also the namespace where the CDI Controller and CDI Importer pods run.
+```shell
+$ mkdir cdi-manifests && cd cdi-manifests
+$ wget https://raw.githubusercontent.com/copejon/containerized-data-importer/kubevirt-centric-readme/manifests/example/golden-pvc.yaml
```

Comment:
should these urls link to _kubevirt/containerized-data-importer_ rather than _jcope_?

Reply:
Right, will fix.